packages/devextreme/js/__internal/ui/chat/messagelist.ts

```diff
@@ -18,8 +18,10 @@ import Widget from '@ts/core/widget/widget';
 import { isElementVisible } from '../splitter/utils/layout';
 import type { MessageGroupAlignment } from './messagegroup';
 import MessageGroup from './messagegroup';
+import TypingIndicator from './typingindicator';
 
 const CHAT_MESSAGELIST_CLASS = 'dx-chat-messagelist';
+const CHAT_MESSAGELIST_CONTENT_CLASS = 'dx-chat-messagelist-content';
```

Comment:
Did I understand correctly that it needs to add typing indicator inside scrollview-content element?
Why we can't to insert typing indicator after scrollview content and not add additional level of dom hierarchy?

Reply:
Simething like on this screenshot <img width="1214" alt="image" src="https://github.com/user-attachments/assets/3d486493-0a91-401c-87d7-4d100f92538f">

Reply:
I'm sure it's methodologically correct to add any content to an element with the dx-scrollview-content class. I don't think it's the right tactic to add other elements to the ScrollView markup. Any such additions should come from the ScrollView component, not the MessageList. 

In addition, ScrollView does not provide any public or even private API for adding anything to its markup.